routes/web.php

```diff
@@ -45,11 +45,12 @@
  */
 Route::get('log-out', ['as' => 'log-out', 'uses' => 'Auth\LoginController@logout']);
 
-Auth::routes();
+// Disable email registration
+Auth::routes(['register' => false]);
 
-Route::get('sign-up', function () {
-    return redirect('register');
-});
+// Route::get('sign-up', function () {
+//     return redirect('register');
+// });
```

Comment:
Do we know why this route was here to begin with? I don't see it referenced anywhere in the code. If it's used by some external link (like from an email), then we should probably leave it active, and instead redirect to `'/'` or something.

Reply:
I imagine it was from an older version of Laravel before the auth routes were included.  I went ahead and removed it since it isn't referenced anywhere in the app.